src/storekit.cs

```diff
@@ -525,34 +545,51 @@ interface SKStoreProductViewController {
 		void LoadProduct (StoreProductParameters parameters, [NullAllowed] Action<bool,NSError> callback);
 	}
 
+	[Mac (11,0), NoTV, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[Model]
+	[Protocol]
+	interface SKStoreProductViewControllerDelegate {
+		[Export ("productViewControllerDidFinish:"), EventArgs ("SKStoreProductViewController")]
+		void Finished (SKStoreProductViewController controller);
+	}
+
 	[NoWatch]
-	[NoMac]
+	[Mac (11,0)]
```

Comment:
why ?
it looks like you added `[NoMac] on all members, so only an empty type will be generated

Reply:
we need the type for:
```
		[Export ("loadProductWithParameters:completionBlock:")][Internal]
		[Async]
		void LoadProduct (NSDictionary parameters, [NullAllowed] Action<bool,NSError> callback);

		[Wrap ("LoadProduct (parameters.GetDictionary ()!, callback)")]
		[Async]
		void LoadProduct (StoreProductParameters parameters, [NullAllowed] Action<bool,NSError> callback);
```
It is a dictionary without those getters setters. So I think is easier to remove the type, and do not expose the Wrapped one.

Reply:
Correction, the strong dictionary is missing values from an older binding, is the better approach.